src/bash.d/01-crew

```diff
@@ -32,12 +37,15 @@ function _crew () {
     install
     list
     postinstall
+    prop
     reinstall
     remove
     search
     sysinfo
+    test
     update
     upgrade
+    upload
```

Comment:
Good call.  This is what happens when we split out these things.  Everything becomes too abstracted.